tools/python/boutiques/bosh.py

```diff
@@ -420,7 +425,7 @@ def bosh_return(val, code=0):
         if runs_as_cli():
             try:
                 print(e.message)
-            except Exception as e:
+            except Exception as ex:
                 print(e)
```

Comment:
need to change to `print(ex)` I think, since you renamed the exception from `e` to `ex`?

Reply:
that was exactly the bug: we want `e` printed, not `ex`. In Python 3, `ex` would just be `object has no attribute message`.